worldengine/draw.py

```diff
@@ -425,8 +395,8 @@ def draw_riversmap_on_file(world, filename):
 
 
 def draw_grayscale_heightmap_on_file(world, filename):
-    img = ImagePixelSetter(world.width, world.height, filename)
-    draw_grayscale_heightmap(world, img)
+    img = PNGWriter.grayscale_from_array(world.elevation['data'], filename, scale_to_range=True)
+    #draw_grayscale_heightmap(world, img)
```

Comment:
To sum this up and finish it:
The image for the grayscale-heightmap is now generated basically by *directly writing the corresponding numpy-array into an image-file*. There is no middleman necessary.